src/homework/api/orders.py

```diff
@@ -1,3 +1,52 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends
+from .contracts import (

```

Comment:
same about relative imports, as in the previous commit

Reply:
Will be fixed!